content/blog/how-to-backup-and-restore-stateful-applications-on-kubernetes-using-kasten-k10-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,660 @@
+gl-sbp-frank-gl1-sstor01 (default)   csi.hpe.com                    Delete          Immediate              true                   56d
+hpe-hdd-storage                      kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  56d
+hpe-nvme-storage                     kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  56d
+hpe-ssd-storage                      kubernetes.io/no-provisioner   Delete          WaitForFirstConsumer   false                  56d
+
+$ kubectl  get volumesnapshotclasses
+NAME                                 DRIVER        DELETIONPOLICY   AGE
+gl-sbp-frank-gl1-sstor01             csi.hpe.com   Delete           56d
+```
+
+[The joint partnership between HPE and Veeam](https://www.kasten.io/kubernetes/resources/blog/kubernetes-backup-with-hpe-csi-and-kasten-k10) supports HPE CSI driver for K8s and Kasten K10 as a data management solution for K8s backup and recovery. The following sections will show you how to install Kasten K10 on the cluster and how to use it with HPE CSI driver for K8s to backup and restore the persistent volumes of the stateful applications running in the cluster using volume snapshots. 
+
+### Prerequisites
+
+Before starting, make sure you meet the following requirements:
+
+* A K8s cluster, being provisioned in HPE GreenLake for Private Cloud Enterprise
+* The kubectl CLI tool, together with the kubeconfig file for accessing the K8s cluster
+* The [Helm](https://helm.sh/docs/intro/install/) CLI tool, version 3.12.1 or later
+* The o﻿ptional mysql CLI tool, for accessing the deployed sample MySQL database service
+
+### Install Kasten K10
+
+Kasten K10 can be deployed on K8s like any other application and it runs in its own namespace. 
+
+F﻿ollowing the [Kasten K10 installation page](https://docs.kasten.io/latest/index.html), Kasten K10 can be installed to the cluster with the following commands using helm:
+
+```shell
+$ helm repo add kasten https://charts.kasten.io/
+$ helm repo update
+
+$ helm install k10 kasten/k10 --namespace=kasten-io --create-namespace
+NAME: k10
+LAST DEPLOYED: Thu Jan 18 22:34:17 2024
+NAMESPACE: kasten-io
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Thank you for installing Kasten\u2019s K10 Data Management Platform 6.5.2!
+
+Documentation can be found at https://docs.kasten.io/.
+
+How to access the K10 Dashboard:
+
+To establish a connection to it use the following `kubectl` command:
+
+`kubectl --namespace kasten-io port-forward service/gateway 8080:8000`
+
+The Kasten dashboard will be available at: `http://127.0.0.1:8080/k10/#/`
+```
+
+With above commands, Kasten K10 is installed to the namespace *'kasten-io'* in the cluster. To validate the installation, typing the following command to watch for the status of all Pods. Helm installs a list of Pods to the namespace. It takes a while before all those Pods start running.  
```

Comment:
Second sentence:
typing the following command to watch the status of all Pods.  (remove the "for").